### DISCUSSION TITLE
LFX Mentorship Mentee Applications for '23 Term 02 (June - August)

### DISCUSSION THREAD

**Discussion by aliok** [cncf/mentoring#988] [Announcements]
Hello everyone,

The LFX mentorship projects for 2023 Term 02 (June - August) are now listed publicly, and can be found here:
https://github.com/cncf/mentoring/tree/main/programs/lfx-mentorship/2023/02-Jun-Aug .

Mentee applications are now open!

Applications will be open until May 23, 24:00 UTC (5:00 PM PDT).

We encourage everyone interested to apply and engage with the project community and mentors.

# Mentee Timeline: June 1st - August 31st

## Applications

* May 10th - May 24th, 2023 24:00 UTC / 5:00 PM PDT (2 weeks): Mentee applications open
* May 24th - May 29th: Application review by the mentors, admission decisions

## Mentorship duration - three months (12 weeks - full-time schedule)

* June 1, 2023 (Week 1): Mentorship program begins with the initial work assignments
* July 12, 2023 (End of Week 6): Midterm mentee evaluations and first stipend payments
* August 23, 2023 24:00 UTC / 5:00 PM PDT (End of Week 12): Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals
* August 31: Last day of term

**Comment by Iupac120** [comment]
I want to participate

**Comment by pristine-akw** [comment]
How do I apply?

**Reply by ashutosh-rath02** [reply]
https://mentorship.lfx.linuxfoundation.org/project/c45cc842-278f-4663-9ff4-deecc3fc040d
You can apply from here!!

**Comment by sanya031** [comment]
Hi I had applied for the program long back and my application status is still pending. Can you please check on my application and confirm the status? 

Thanks 
Sanya

**Reply by nate-double-u** [reply]
There are still a couple of projects that haven't made their selections: [Kubescape](https://cloud-native.slack.com/archives/CGPK98JNQ/p1685479867096649) & Meshery. We hope to have them soon.
